Backoffice/Collector/RoleCollector.php

```diff
@@ -77,8 +77,8 @@ public function getRolesByType($type)
     {
         $selectedRoles = array();
 
-        foreach ($this->roles as $role =>  $translation) {
-            if (preg_match('/role_access_([^_]*_)?' . $type . '/i', $role)) {
+        foreach ($this->roles as $role => $translation) {
+            if (preg_match('/role_access_([^_]*_)?' . $type . '$/i', $role)) {
```

Comment:
you could also add the `^` character at the begining of the regex, so you will be sure that nothing more is matched